docs/integration/postgresql.md

```diff
@@ -1,9 +1,55 @@
 title: PostgreSQL Monitoring Integration
 description: Comprehensive view of your database’s health and performance with Sematext PostgreSQL monitoring integration. Our infrastructure monitoring tools provide real-time visibility into the performance and availability of various PostgreSQL databases metrics. Use built-in reports and dashboards, and identify and investigate PostgreSQL database server issues before they become incidents
 
-## Integration
+Use the Sematext PostgreSQL Monitoring integration to check on your PostgreSQL metrics. From number of rows and connections to index and function-specific metrics, all the relevant stats are collected and displayed in built-in dashboards:
 
-- Instructions: [https://apps.sematext.com/ui/howto/postgresql/overview](https://apps.sematext.com/ui/howto/postgresql/overview)
+<img
+  class="content-modal-image"
+  alt="PostgreSQL Monitoring Overview"
+  src="../../images/agents/postgresql_monitoring_overview.png"
+  title="PostgreSQL Monitoring Overview"
+/>
+
+Be sure to check out the [PostgreSQL Logs integration](./postgresql-logs.md) as well, to get a complete view on PostgreSQL. For example, if you see a spike in number of rows fetched, you can check logs to see which queries fetch too many rows. A concrete example of doing that can be found in [this article](https://sematext.com/blog/postgresql-slow-queries/).
+
+## Agent Install
+
+You can find the [complete instructions](https://apps.sematext.com/ui/howto/postgresql/overview) in the Integrations screen of your Monitoring App, but the basic steps are:
+
+- install the Sematext Agent
+- create a user for the Sematext Agent to use
+- configure Sematext Agent to connect to the PostgreSQL host and port (localhost:5432 by default), using your App token and the newly created user's credentials
+
+## Important RabbitMQ Metrics to Watch
```

Comment:
You mean PostgreSQL?

Reply:
Yes, that's exactly what I meant :) Thanks for catching it!